pagopa/io-app
# feat(Firma con IO): [SFEQS-1462] Update Error component introducing retry and assistance CTA

### This PR depends on #4527 

## Short description
This PR updates the error component with retry and assistance CTA.

![Simulator Screen Shot - iPhone 14 - 2023-04-12 at 09 43 48](https://user-images.githubusercontent.com/49342144/231388819-38d5be13-8b61-4c5d-87c4-7afdbc84c8a1.png)

## List of changes proposed in this pull request
- Update ErrorComponent
- Update test

## How to test
Using io-dev-server a light way to try these changes is to edit SuccessComponent.tsx returning the GenericErrorComponent with retry and/or assistance prop set. Btw it should be enough that tests solve correctly.

```typescript
return (
    <GenericErrorComponent
      title={I18n.t("features.fci.errors.generic.default.title")}
      subTitle={I18n.t("features.fci.errors.generic.default.subTitle")}
      onPress={() => dispatch(fciEndRequest())}
      testID="PollingErrorComponentTestID"
      retry={true}
      assistance={true}
    />
  );
```
